addons/web/static/src/views/fields/list_stat/contact_stat.js

```diff
@@ -0,0 +1,23 @@
+import { registry } from "@web/core/registry";
+import { standardFieldProps } from "../standard_field_props";
+import { _t } from "@web/core/l10n/translation";
+import { Component } from "@odoo/owl";
+
+export class ContactStatField extends Component {
+    static template = "web.ContactStatField";
+    static props = {
+        ...standardFieldProps,
+    };
+
+    get list() {
+        return this.props.record.data[this.props.name] || [];
+    }
+}
+
+export const contactStatField = {
+    component: ContactStatField,
+    displayName: _t("Contact Stat"),
```

Comment:
Maybe we could be more specific as it supports only list view, so "List Contact Statistics" ?